config/fgcmBuildStars.py

```diff
@@ -8,20 +8,18 @@
 # The mapping from HSC short filter name to FGCM "band".  This mapping
 # ensures that stars observed in HSC-R + HSC-R2 can be matched, and
 # HSC-I + HSC-I2 can be matched.
-config.filterMap = {'g': 'g', 'r': 'r', 'r2': 'r', 'i': 'i', 'i2': 'i',
-                    'z': 'z', 'y': 'y',
-                    'N387': 'N387', 'N816': 'N816', 'N921': 'N921',
-                    'N1010': 'N1010'}
+from lsst.obs.hsc.hscFilters import HSC_FILTER_DEFINITIONS
+config.physicalFilterMap = HSC_FILTER_DEFINITIONS.physical_to_band
 config.primaryBands = ('i', 'r', 'g', 'z', 'y', 'N387', 'N816', 'N921', 'N1010')
 config.doSubtractLocalBackground = True
 
 config.fgcmLoadReferenceCatalog.refObjLoader.ref_dataset_name = 'ps1_pv3_3pi_20170110'
 # This is the mapping from HSC short filter name to the closest PS1 filter from
 # the reference catalog.  PS1 does not have narrow-band filters.
-config.fgcmLoadReferenceCatalog.refFilterMap = {'g': 'g', 'r': 'r', 'r2': 'r',
-                                                'i': 'i', 'i2': 'i', 'z': 'z', 'y': 'y',
-                                                'N387': 'g', 'N816': 'i', 'N921': 'z',
-                                                'N1010': 'y'}
+config.fgcmLoadReferenceCatalog.refFilterMap = {'HSC-G': 'g', 'HSC-R': 'r', 'HSC-R2': 'r2',
```

Comment:
These refcat filterMaps should be doing this, I'd think?

```
config.fgcmLoadReferenceCatalog.filterMap = {}
filterMapFile = os.path.join(os.path.dirname(__file__), "filterMap.py")
config.fgcmLoadReferenceCatalog.load(filterMapFile)
```

Refcat filter maps should be band->refcat filter. Which means for PS1 you wouldn't need anything except the narrow bands. Unless this is different from the `loadReferenceObjects` `filterMap`s?